frontend/src/pages/home/home.module.css

```diff
@@ -1,29 +1,100 @@
-.section{
-    /* border:2px solid red; */
-    height:100vh;
-    margin:0 1rem 0 1rem;
-}
-.header{
+.hero{
+    border-top:1px solid var(--main-color);
+ 
+    margin:0rem 1rem;
     display: flex;
-    flex-direction: row;
-    border-bottom:1px solid var(--main-color);
-    margin-top: 2.5rem;
-    justify-content:space-between;
-    align-items: baseline;
+    flex-direction: column;
+    position: relative;
 }
-.navbar{
+.hero_text{
+    padding:2.5rem 0;
     display: flex;
-    flex-direction: row;
-    justify-content: flex-end;
+    flex-direction: column;
+    justify-content: center;
+    line-height: 3rem;
+}
+.text{
+    text-align:center;
+}
+.text h1{
+    text-align: center;
+}
+.text p{
+    text-align: center;
+}
+.weekly{
+    background-color: var(--home-button-color);
+    color:#fff;
+    border-radius:30px;
+    padding:0 1rem;
+    width:160px;
 }
 
-.linkContainer{
-    margin-right: 1rem;
-    text-decoration: none;
-    color:var(--main-color)
+.half{
```

Comment:
In the Figma file this image has its center between the text box to the left and the image to the right, and the bottom is aligned with the thin line on the page. On my screen the half circle isn't aligned in this way. The bottom is slightly below the line and the half circle is positioned more to the right.

Reply:
have solved the problem